internal/cloudapi/v2/handler.go

```diff
@@ -464,7 +464,13 @@ func (h *apiHandlers) getJobIDComposeStatus(jobId uuid.UUID) (ComposeStatus, err
 			ImageStatuses: &buildJobStatuses,
 			KojiStatus:    &KojiStatus{},
 		}
+		/* #nosec G115 */
```

Comment:
I'd also add a comment for the rationale here … but no showstopper I think

Reply:
Actually the description is right below, it could underflow so we have to ignore it, but the underflow is checked right after.

Reply:
… right - 2 LOC - that was beyond my attention span 🫣 😁